cypress/e2e/forgotPassword.cy.js

```diff
@@ -0,0 +1,28 @@
+import forgotPasswordPage from '../support/page_objects/forgotPasswordPage';
+import * as page from '../fixtures/URLs.json';
+
+describe('ForgotPassword page spec', () => {
+  it('should show a validation error when the email field is left blank', () => {
+    cy.visit(page.forgotPassword);
+    forgotPasswordPage.confirmResetButton().click();
+    cy.get('input[name="email"]').then(($input) => {
+      expect($input[0].validationMessage).to.eq('Please fill in this field.');
+    });
+  });
+  it('should redirect the user to a success page when filling the valid email form and clicking "Confirm Reset" button', () => {
+    cy.visit(page.forgotPassword);
+    console.log('Login Email:', Cypress.env('loginEmail'));
+    forgotPasswordPage.emailInput().type(Cypress.env('loginEmail'));
+    forgotPasswordPage.confirmResetButton().click();
+    cy.url().should('include', '/login/confirm');
+  });
+
+  it('should show an error message when an unregistered email is submitted', () => {
+    cy.visit(page.forgotPassword);
+    const unregisteredEmail = 'enregistered@example.com';
+    forgotPasswordPage.emailInput().type(unregisteredEmail);
+    forgotPasswordPage.confirmResetButton().click();
+    cy.url().should('include', '/login/forgot-password');
+    cy.get('p').contains('User does not exist. Please sign uo');
```

Comment:
```js
    cy.get('p').contains(/User does not exist. Please sign up/i');
```
Changing the string to a RegEx can keep the test from breaking if we make small edits to the user message in future :)